Tools/TestWebKitAPI/Tests/WebKit/AdvancedPrivacyProtections.mm

```diff
@@ -1009,14 +1009,15 @@ HTTPServer server({
 TEST(AdvancedPrivacyProtections, Canvas2DQuirks)
 {
     using namespace TestWebKitAPI;
-    const auto script = @"let canvas = document.createElement(\"canvas\"); let ctx = canvas.getContext(\"2d\"); ctx.fillText(\"<@nv45. F1n63r,Pr1n71n6!\", 10, 40); canvas.toDataURL();";
+    auto* script = @"let canvas = document.createElement(\"canvas\"); canvas.width = 280; canvas.height = 60; let ctx = canvas.getContext(\"2d\"); ctx.fillText(\"<@nv45. F1n63r,Pr1n71n6!\", 10, 40); canvas.toDataURL();";
```

Comment:
```suggestion
    auto *script = @"let canvas = document.createElement(\"canvas\"); canvas.width = 280; canvas.height = 60; let ctx = canvas.getContext(\"2d\"); ctx.fillText(\"<@nv45. F1n63r,Pr1n71n6!\", 10, 40); canvas.toDataURL();";
```

Reply:
Looking at the current codebase, it seems like `auto*` is more common than `auto *`.

Reply:
Hm... I think we generally use `auto *` in Objective-C files whereas we use `auto* ` in C++ files.

Reply:
Yes, we use `Type *` instead of `Type*` in Objective C code.

Reply:
Ah, thanks for clarifying